pkg/config/configuration.go

```diff
@@ -452,3 +439,133 @@ func (c Configuration) EnabledFeatures() []string {
 	}
 	return features[:i]
 }
+
+// GetTracingSpec returns the tracing spec.
+// It's a short-hand that includes nil-checks for safety.
+func (c Configuration) GetTracingSpec() TracingSpec {
+	if c.Spec.TracingSpec == nil {
+		return TracingSpec{}
+	}
+	return *c.Spec.TracingSpec
+}
+
+// GetMTLSSpec returns the mTLS spec.
+// It's a short-hand that includes nil-checks for safety.
+func (c Configuration) GetMTLSSpec() MTLSSpec {
+	if c.Spec.MTLSSpec == nil {
+		return MTLSSpec{}
+	}
+	return *c.Spec.MTLSSpec
+}
+
+// GetMetricsSpec returns the metrics spec.
+// It's a short-hand that includes nil-checks for safety.
+func (c Configuration) GetMetricsSpec() MetricSpec {
+	if c.Spec.MetricSpec == nil {
+		return MetricSpec{}
+	}
+	return *c.Spec.MetricSpec
+}
+
+// GetAPISpec returns the API spec.
+// It's a short-hand that includes nil-checks for safety.
+func (c Configuration) GetAPISpec() APISpec {
+	if c.Spec.APISpec == nil {
+		return APISpec{}
+	}
+	return *c.Spec.APISpec
+}
+
+// GetLoggingSpec returns the Logging spec.
+// It's a short-hand that includes nil-checks for safety.
+func (c Configuration) GetLoggingSpec() LoggingSpec {
+	if c.Spec.LoggingSpec == nil {
+		return LoggingSpec{}
+	}
+	return *c.Spec.LoggingSpec
+}
+
+// GetLoggingSpec returns the Logging.APILogging spec.
+// It's a short-hand that includes nil-checks for safety.
+func (c Configuration) GetAPILoggingSpec() APILoggingSpec {
+	if c.Spec.LoggingSpec == nil || c.Spec.LoggingSpec.APILogging == nil {
+		return APILoggingSpec{}
+	}
+	return *c.Spec.LoggingSpec.APILogging
+}
+
+// ToYAML returns the Configuration represented as YAML.
+func (c *Configuration) ToYAML() (string, error) {
```

Comment:
See a func with same name but for `ConfigurationSpec`, both are used only one time, I think no need to make it as a func.

Reply:
They are primarily used for debugging and testing.